src/contributors/roshangyawali.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "roshan gyawali"
+username: "roshan00017"
+image: "rg.jpg"
+
+---
+I am Roshan Gyawali
```

Comment:
Thank you roshan for pull request but please follow the docs, add you image inside src/image